### PR TITLE
feat: add DurationFormatterUtilTest

### PR DESCRIPTION
**Pull request by KarpovIvan**
## Description

task: [24657](https://github.com/camunda/camunda/issues/24657)

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #

### CODE REVIEW

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    // given
+    final long durationInMs = 4000000000L; // 46 days, 6 hours, 40 minutes
+
+    // when
+    final String result =
+        DurationFormatterUtil.formatMilliSecondsToReadableDurationString(durationInMs);
+
+    // then
+    assertThat(result).isEqualTo("1mo 2wks 2d 7h 6min 40s");
+  }
```
Comment (RomanJRW) [inline optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java:27]: nice 👍

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+    // given
+    durationInMs = -1000;
+
+    // when
+    result = DurationFormatterUtil.formatMilliSecondsToReadableDurationString(durationInMs);
+
+    // then
+    assertThat(result).isEqualTo("-");
+  }
```
Comment (RomanJRW) [inline optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java:48]:
```suggestion
  @ParameterizedTest
  @ValueSource(ints = {0, -1000})
  void shouldReturnDashForZeroOrNegativeDuration(final int durationInMs) {
    // when
    String result = DurationFormatterUtil.formatMilliSecondsToReadableDurationString(durationInMs);

    // then
    assertThat(result).isEqualTo("-");
  }
```

NABD but I like this very succinct way of testing with parameters

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    // given
+    duration = 0;
+
+    // when
+    result = DurationFormatterUtil.formatDuration(unitString, duration);
+
+    // then
+    assertThat(result).isEmpty();
+  }
+}
```
Comment (RomanJRW) [inline optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java:104]: this could also be parameterized as above

**Review by RomanJRW — APPROVED** [review]
Nice progress. Pre-approving with some suggestions 🙂

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java (hunk 2 above)
Comment (KarpovIvan) [reply optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java]: fixed

**Pull request merged**